test/step-definitions/template.step.ts

```diff
@@ -46,24 +51,36 @@ defineFeature(feature, (test): void => {
     })
   })
 
-  test('Hello', ({ given, when, then }): void => {
+  test('CreateThirdpartyTransactionRequests', ({ given, when, then }): void => {
```

Comment:
let extract this out from `template` file to one dedicated to `transactionRequests`